pkg/oc/graph/kubegraph/edges.go

```diff
@@ -33,6 +38,12 @@ const (
 	ReferencedServiceAccountEdgeKind = "ReferencedServiceAccount"
 	// ScalingEdgeKind goes from HorizontalPodAutoscaler to scaled objects indicating that the HPA scales the object
 	ScalingEdgeKind = "Scaling"
+	// TriggersDeploymentEdgeKind points from DeploymentConfigs to ImageStreamTags that trigger the deployment
+	TriggersDeploymentEdgeKind = "TriggersDeployment"
```

Comment:
this was duplicated from appsgraph because of import cycles... we should probably move this into some common package

Reply:
follow up, don't want to pollute this PR